product_docs/docs/pgd/5/rel_notes/pgd_5.4.0_rel_nodes.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: "EDB Postgres Distributed 5.4.0 release notes"
+navTitle: "Version 5.4.0"
+---
+
+Released: xx xxx 2024
+
+EDB Postgres Distributed version 5.4.0 is a minor version of EDB Postgres Distributed.
+
+!!! Important Recommended upgrade
+We recommend that all users of PGD 5 upgrade to PGD 5.4. See [PGD/TPA upgrades](../upgrades/tpa_overview) for details.
+!!!
+
+## Highlights of EDB Postgres Distributed 5.4.0
+
+* Group Commit and Parallel Apply are now compatible with each other.
+
+## Compatibility
+
+!!! Note EDB Server version compatibility
+This version requires the recently released Postgres versions 14.10, 15.4,
+or 16.1 (or later) of EDB Postgres Advanced Server or EDB Postgres Extended
+Server. No such restrictions exist for PostgreSQL Server.
+
+Package managers on Debian, RHEL, and SLES pull in the required EDB Postgres Advanced Server or EDB Postgres Extended 
+upgrades with an upgrade of EDB Postgres Distributed.
+!!!
+
+## Features
+
+| Component | Version | Description  | Addresses |
+|-----------|---------|-------------|-----------|
+| PGD       | 5.4.0   | Group Commit and Parallel Apply can work together with a dedicated writer for Group Commit. | | /* TBC (To Be Committed) */
```

Comment:
Reviewed docs and removed Known Issue saying that.

Reply:
@djw-m we are no longer releasing the parallel apply + group commit work since bugs have been discovered. please revert those changes, it will still be a known issue with this release

Reply:
Reverting as best I can.

Reply:
Removed the release note line too.